### PR TITLE
QA Observations Bugfix/FOUR-13285: The color of the select list is different and is not centered after saving the process

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
- The color of the select list was fixed

## How to Test
- Go to Process Catalogue
- Click on some Available Process Category
- Click on Any Process without Icon assigned
- Open Edit in Launchpad
- Dropdown must show "Select Icon"
- Save modal window without make changes
- Open again Edit in Launchpad
- Dropdown "Select Icon" must have the same color, size and shape

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13285

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-cefaa9ce76.engk8s.processmaker.net

File: resources/js/components/shared/IconDropdown.vue [hunk 1]
```diff
@@ -160,9 +160,14 @@ $multiselect-height: 33px;
   }
 
   .multiselect__single {
-    font-size: 15px;
+    font-size: 16px;
```
Comment (pmPaulis) [inline resources/js/components/shared/IconDropdown.vue:163]: Please apply this same size in the Select chart

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/js/components/shared/IconDropdown.vue (hunk 1 above)
Comment (pmPaulis) [reply resources/js/components/shared/IconDropdown.vue:163]: image

Comment (CarliPinell) [reply resources/js/components/shared/IconDropdown.vue:163]: Required changes were applied:
image

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-cefaa9ce76.engk8s.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**